### DISCUSSION TITLE
is there any way to add pages?

### DISCUSSION THREAD

**Discussion by feyradio** [saicaca/fuwari#227] [Q&A]
i am new to astro and have no idea how to work this around. because even creating them manually doesn't do much, because I cannot add them to the navigation. adding a new one just breaks the entire thing.

**Comment by saicaca** [comment]
Just create a new `.astro` file under the pages directory, and the route will be generated based on the file path.
You may need to wrap the page content inside the `<MainGridLayout>` tag. The about page `about.astro` can serve as a reference
